clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/ApplicationEventProcessor.java

```diff
@@ -569,6 +571,8 @@ private void process(final SeekUnvalidatedEvent event) {
                 metadata.currentLeader(event.partition())
             );
             subscriptions.seekUnvalidated(event.partition(), newPosition);
+            requestManagers.commitRequestManager.ifPresent(commitRequestManager ->
+                commitRequestManager.allConsumed(subscriptions.allConsumed()));
```

Comment:
Why is `allConsumed()` called here again, but this time with the data from `SubscriptionState.allConsumed()`. Please add some comments to explain. Thanks.